Assignment-1/services/src/moreorders.go

```diff
@@ -0,0 +1,26 @@
+package src
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+)
+
+func MoreOrders() bool {
+	reader := bufio.NewReader(os.Stdin)
```

Comment:
why not use fmt.scan methods. fmt.scanf fmt.scanln:

Reply:
those methods return int value which was not required